analyzers/tests/SonarAnalyzer.Test/TestCases/UnusedPrivateMember.CSharp9.cs

```diff
@@ -168,3 +169,121 @@ public class ClassWithPrintMembers
         }
     }
 }
+
+// https://github.com/SonarSource/sonar-dotnet/issues/9379
+namespace Repro_9379
+{
+    public static class Program
+    {
+        public static void Method()
+        {
+            var instance = CreateInstance<ClassInstantiatedThroughReflection>();
+
+            A classViaReflection = new();
+            InitValue(classViaReflection); 
+        }
+
+        public static T CreateInstance<[DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.All)] T>() =>
+            (T)Activator.CreateInstance(typeof(T), 42);
+
+        public static void InitValue<[DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.All)] T>(T value) { }
+
+        private class A
+        {
+            private bool a = true; // Noncompliant FP: type argument is inferred and ArgumentList is not present on syntax level
```

Comment:
Can you point to the place where the FP originates from. It is hard to see that line 183 is related to this code.